script/lib/install-script-runner-dependencies.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const childProcess = require('child_process');
+
+const CONFIG = require('../config');
+
+process.env.ELECTRON_CUSTOM_VERSION = CONFIG.appMetadata.electronVersion;
```

Comment:
https://github.com/atom/atom/commit/9e8b0b14faea0b2d08ffadbe44272c386749dd53

Reply:
The problem with 'just to be safe' is it ends up with a whole bunch of code that may or may not be necessary, but is there 'just to be safe' and no-one knows if removing it would break anything or not. I would say: if it is necessary, add a comment explaining why. If not, remove it.

And if it is kept, it should be consolidated with the other usages.

Reply:
```suggestion
```